database/migrations/20200102171324-create-event.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("Events", {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4
+      },
+      type: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      img: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      title: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      desc: {
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      creator: {
+        type: Sequelize.UUIDV4,
+        allowNull: true
```

Comment:
Creator is a foreign key that references user id, I am not sure it should allow null